src/main/java/uk/gov/pay/ledger/transaction/state/PaymentState.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.pay.ledger.transaction.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Arrays.stream;
+import static java.util.List.of;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.CAPTURABLE;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.CREATED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.ERROR_GATEWAY;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_CANCELLED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_EXPIRED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_REJECTED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.STARTED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.SUBMITTED;
+
+public enum PaymentState {
+
+    IN_PROGRESS(of(CREATED, STARTED, SUBMITTED, CAPTURABLE), "In progress"),
+    SUCCESS(of(TransactionState.SUCCESS), "Success"),
+    DECLINED(of(FAILED_REJECTED), "Declined"),
+    TIMED_OUT(of(FAILED_EXPIRED), "Timed out"),
+    CANCELLED(of(FAILED_CANCELLED, TransactionState.CANCELLED), "Cancelled"),
+    ERROR(of(TransactionState.ERROR, ERROR_GATEWAY), "Error");
```

Comment:
likewise here

Reply:
Has to be qualified as it conflicts with the enum of same name